sdk/identity/ci.yml

```diff
@@ -71,12 +71,29 @@ extends:
     LiveTestStages:
       - template: /eng/pipelines/templates/stages/archetype-sdk-tests-isolated.yml
         parameters:
+          PreSteps:
+            - task: AzureCLI@2
+              displayName: Set OIDC variables
+              env:
+                ARM_OIDC_TOKEN: $(ARM_OIDC_TOKEN)
+                ARM_CLIENT_ID: $(ARM_CLIENT_ID)
+                ARM_TENANT_ID: $(ARM_TENANT_ID)
+              inputs:
+                azureSubscription: azure-sdk-tests
```

Comment:
See my other comment https://github.com/Azure/azure-sdk-for-java/pull/42635/files#r1819384979 but you should not need this PreStep and as it currently setup it will not work because we need to have the subscription match the deployment.